### DISCUSSION TITLE
Status Bar Not Updating

### DISCUSSION THREAD

**Discussion by sshaheen** [catppuccin/tmux#285] [Help] [answered]
Hello,

My status bar is not updating in tmux.
![image](https://github.com/user-attachments/assets/e4fd531c-c0b9-442d-ad06-ba37abb4d676)
I have CaskaydiaCove Nerd Font on my Ubuntu shell

**Comment by vdbe** [comment, marked as answer]
I guess you come from the dracula theme/tried out the dracula theme before this?

catppuccin/tmux and dracula/tmux have a naming conflict in tpm.
removing the `tmux` plugin in `~/.tmux/plugins/tmux` and installing again via tpm should fix it.

**Reply by sshaheen** [reply]
Awesome! Yes I was trying dracula before this. Thanks so much! :)
